play/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Right On",
```

Comment:
Do we need `!` at the end?

Reply:
Good point. This follows `central` and `host` but I can update the three of them to include it.